spyder/plugins/editor/widgets/editor.py

```diff
@@ -1161,6 +1163,13 @@ def set_convert_eol_on_save_to(self, state):
     def set_focus_to_editor(self, state):
         self.focus_to_editor = state
 
+    def set_run_cell_func(self, state):
+        """If `state` is `True`, run_cell() will be used for code cells."""

```

Comment:
``True`` is a literal, not a name, so it should have double backticks. Also, ``run_cell`` is a name so it should have single backticks.